pacai/agents/capture/reflex.py

```diff
@@ -51,7 +51,7 @@ def evaluate(self, gameState, action):
         features = self.getFeatures(gameState, action)
         weights = self.getWeights(gameState, action)
 
-        return features * weights
+        return sum(features[feature] * weights[feature] for feature in features)
```

Comment:
Not a fan of this return statement. Instead, create a variable name for it and return that variable name.